pypeit/images/scienceimage.py

```diff
@@ -330,10 +337,11 @@ def build_ivar(self):
         # Convenience
         detector = self.spectrograph.detector[self.det-1]
         # For RN variations by Amp
-        datasec_img = self.spectrograph.get_datasec_img(filename=self.filename,
-                                                     det=self.det)
+        #embed(header='333 of sciencimage')
+        #datasec_img = self.spectrograph.get_datasec_img(filename=self.filename,
+        #                                             det=self.det)
         # Generate
```

Comment:
See my comment above. All these operations should be in one image processing routine. These steps have nothing to do with the image being a science image.

Reply:
Future PR